pkg/agent/tunnelserver/tunnelserver.go

```diff
@@ -237,6 +238,26 @@ func (t *tunnelServer) GitSSHSignature(ctx context.Context, message *tunnel.Mess
 	return &tunnel.Message{Message: string(out)}, nil
 }
 
+func (t *tunnelServer) LoftConfig(ctx context.Context, message *tunnel.Message) (*tunnel.Message, error) {
```

Comment:
I think we also need an implemenation of this in tunnelserver/proxyserver 🤔

Reply:
👀